debugger/terminal/terminal_windows.go

```diff
@@ -0,0 +1,13 @@
+// +build windows
+
+package terminal
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+)
+
+func ConnectTerm(ctx context.Context, addr string) error {
+	return errors.New("Debugger not supported on Windows yet")
```

Comment:
```suggestion
	return errors.New("debugger not supported on Windows yet")
```

Nit: Error strings should start with lower-case